crates/compiler/can/src/expr.rs

```diff
@@ -1016,9 +1016,11 @@ pub fn canonicalize_expr<'a>(
                 (expr, output)
             }
         }
-        ast::Expr::Var { module_name, ident } => {
-            canonicalize_var_lookup(env, var_store, scope, module_name, ident, region)
-        }
+        ast::Expr::Var {
+            module_name,
+            ident,
+            suffixed: _, // TODO should we use suffixed here?
```

Comment:
I wasn't sure about the canonicalise parts. I think we should desugar all of the suffixed `Var`s before can I think it should be safe to just ignore this field.